### ISSUE TITLE
The component `Tempest\View\Components\Form` is missing the attribute `enctype` when it's declared

### ISSUE THREAD

**Issue by yassiNebeL** [tempestphp/tempest-framework#381]
Regarding this issue, I was working on the issue #370 the file uploads, after many attempts, I couldn't find the file uploaded from a form submission, of course I did added the attribute `enctype="multipart/form-data"` but the `$_FILES` looks empty, then I went on to inspect the code source, there I found that the attribute `enctype` wasn't interpreted. So, here is the `Form` class:

```php
    public function render(GenericElement $element, ViewRenderer $renderer): string
    {
        $action = $element->getAttribute('action');
        $method = $element->getAttribute('method') ?? 'post';

        return <<<HTML
<form action="{$action}" method="{$method}">
    <x-slot />
</form>
HTML;
    }
 ```
Could someone fix this issue, I believe this is easy to fix.

**Comment by yassiNebeL** [comment]
@aidan-casey, could you please tag this issue as a bug ( _I don't have this ability yet_ ), actually I spent 30mn to understand why the server-side request doesn't include the file I uploaded, then I inspect the code, and I found that the `enctype` was missing. Worry not, I'll send a PR, but we need to encourage ppl in the community to contribute to the code.

**Comment by aidan-casey** [comment]
Hey, @yassiNebeL, thanks for making us aware of this. Can you submit a PR with a failing test?

**Comment by yassiNebeL** [comment]
```php
final readonly class Form implements ViewComponent
{
...
    public function render(GenericElement $element, ViewRenderer $renderer): string
    {
        $attributes = $element->getAttributes();

        $attrString = implode(' ', array: array_map(
            fn($key, $value) => is_bool($value)
                ? htmlspecialchars($key)
                : sprintf('%s="%s"', htmlspecialchars(ltrim($key, ':')), htmlspecialchars($value)
            , array_keys($element->getAttributes()), $element->getAttributes()
        ));
        
        return <<<HTML
<form {$attrString}>
    <x-slot />
</form>
HTML;
```
With this, attributes are implemented dynamically in the component, the expected outcome for a test would be that the form correctly renders all provided attributes, such as action, enctype, class, and any others passed in.

Let me know if I missed smth please

**Comment by brendt** [comment]
Yeah this should be fixed with #450 I think
